OrderApp/View controllers/CategoryTableViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CategoryTableViewController.swift
+//  OrderApp
+//
+//  Created by JeongTaek Han on 2023-01-27.
+//
+
+import UIKit
+
+@MainActor
+class CategoryTableViewController: UITableViewController {
+    
+    private let restautantController = RestaurantController.shared
+    private var categories: [String] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+    }
+    
+    private func configureUI() {
+        Task {
+            do {
+                let categories = try await restautantController.fetchCategories()
+                updateUI(with: categories)
+            } catch {
+                displayError(error, title: "Failed to Fetch Categories")
+            }
+        }
+    }
```

Comment:
@chatcody Can you find risk on this code?

Reply:
@chatcody why do you think add guard statement to the `configureUI`? because restaurantController's type is not optional so i think guard statement don't need it